definitions/divorce/json/CaseField/CaseField-general-referral-nonprod.json

```diff
@@ -84,5 +84,23 @@
     "Label": "General referral decision date",
     "FieldType": "Date",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "20/10/2020",
+    "CaseTypeID": "DIVORCE",
+    "ID": "GeneralReferralPayment",
+    "Label": "How will payment be made?",
+    "HintText": "How will payment be made?",
```

Comment:
This will display the same text twice

Reply:
I think I have copied and pasted what you had :)